builtin/mainmenu/tab_settings.lua

```diff
@@ -203,9 +203,22 @@ local function formspec(tabview, name, tabdata)
 		"label[4.25,3.45;" .. fgettext("Screen:") .. "]" ..
 		"checkbox[4.25,3.6;cb_autosave_screensize;" .. fgettext("Autosave Screen Size") .. ";"
 				.. dump(core.settings:get_bool("autosave_screensize")) .. "]" ..
-		"box[8,0;3.75,4.5;#999999]" ..
-		"checkbox[8.25,0;cb_shaders;" .. fgettext("Shaders") .. ";"
-				.. dump(core.settings:get_bool("enable_shaders")) .. "]"
+		"box[8,0;3.75,4.5;#999999]"
+
+	local video_driver = core.settings:get("video_driver")
+	local shaders_supported = video_driver == "opengl"
```

Comment:
Just to be sure, so that means that shaders are not supported at all on Android...?

Reply:
NM... You say that in the doc section below.

Reply:
@lhofhansl As I wrote in https://github.com/minetest/minetest/issues/2060#issuecomment-401551510, shaders aren’t loaded for GLES. They aren’t GLES-compatible currently anyway, that’s what I try to fix in #7514, but for some reason normal map generation doesn’t work.